edgelet/management/src/apis/identity_api.rs

```diff
@@ -63,6 +63,7 @@ where
 
         let query = ::url::form_urlencoded::Serializer::new(String::new())
             .append_pair("api-version", &api_version.to_string())
+            .append_pair("type", "aziot")
```

Comment:
These are code-gen and shouldn't be changed

Reply:
Fixed